tests/conformance/scripts/run-tests/types.ts

```diff
@@ -0,0 +1,22 @@
+import { TestFeatures } from "../../variants/types";
+
+export type IndexedReadTestResult = {
+  schemas: SerializableMcapRecord[];
+  channels: SerializableMcapRecord[];
+  messages: SerializableMcapRecord[];
+  statistics: SerializableMcapRecord[];
```

Comment:
Do we ever expect to have more than 1 statistics record?

Reply:
if a writer accidentally puts one there, sure.